ssi-script.md

```diff
@@ -0,0 +1,122 @@
+# Script for SSI Fellowship Application
+
+Slides can be found here :point_right: <https://www.dropbox.com/s/js4uoy8stymc6bo/SGibson_SSI_2020_Fellowship_Application.pdf?dl=0>
+
+#### Slide 1
+
+Hi!
+My name is Sarah Gibson and welcome to my 2020 SSI Fellowship application.
+
+#### Slide 2
+
+Here's a little bit about me.
+
+I'm a Research Data Scientist at The Alan Turing Institute.
```

Comment:
Perhaps mention the Turing is the UK's national institute for Data Science and AI, and a sentence or to about the type of work you do in REG?

Reply:
Added in commit [`2201fff`](https://github.com/sgibson91/ssi-fellowship/pull/1/commits/2201fff46732e17007214b4ae59e7e70821218ff)